interfaces/repo.go

```diff
@@ -656,3 +656,35 @@ func (r *Repository) RemoveSnap(snapInfo *snap.Info) error {
 
 	return nil
 }
+
+// DisconnectSnap disconnects all the connections to and from a given snap.
+//
+// The return value is a list of snap.Info's that were affected.
```

Comment:
Why do we need to return this?  We can only have one possible connected snap, the one that is currently active with the given name.